[13.0][IMP] edi_backend_partner: modify backend view

JaumeBforgeFlow opened the pull request:
Right now when configuring a partner for a backend it appears right next to the backend type when saving. Now the partner will be shown in the next line.

LoisRForgeFlow reviewed (commented):
Thanks for taking care of this! In my opinion it would be better placed below and with the label as any other field.

JaumeBforgeFlow committed:
[IMP] edi_backend_partner: modify backend view

JaumeBforgeFlow commented:
Hi @LoisRForgeFlow, i've updated the code following your feedback and moved the partner field. 

![image](https://user-images.githubusercontent.com/98742635/209117653-7a0ada7e-940a-468f-9452-96c3f70e7a96.png)

MiquelRForgeFlow commented:
FYI https://github.com/OCA/edi/pull/657 exists.

etobella commented:
/ocabot merge patch
